research/src/components/checkbox-anatomy.js

```diff
@@ -8,13 +8,8 @@ export default () => {
         <part name="label">
```

Comment:
Should the line above (`<host name="checkbox">`) have name changed from `checkbox` to `oui-checkbox`?

Reply:
yeah it should, thanks